test/gpu/native/studies/shoc/shoc-sort.skipif

```diff
@@ -1 +1,35 @@
-CHPL_GPU==cpu
+#!/bin/bash
+# Suppress ROCm6 shoc-sort failures: https://github.com/chapel-lang/chapel/issues/26669
+check_conditions() {
+    chpl_gpu=$(printenv CHPL_GPU)
+    chpl_test_perf_label=$(printenv CHPL_TEST_PERF_LABEL)
+
+    # Skip test in CPU-as-device mode
+    if [[ "$chpl_gpu" == "cpu" ]]; then
+        return 1
+    fi
+
+    # Check if CHPL_GPU is 'amd' and CHPL_TEST_PERF_LABEL is set 
+    if [[ "$chpl_gpu" != "amd" || -z "$chpl_test_perf_label" ]]; then
+        return 1
+    fi
+
+    chpl_home=$(printenv CHPL_HOME)
+    if [[ -z "$chpl_home" ]]; then
+        return 1
+    fi
```

Comment:
Now I am curious about this change and why it was needed. But it is not a showstopper at all.

Reply:
Well I forgot that `start_test` fails if `CHPL_HOME` is not set so I added this redundant check.

Reply:
This one is technically a redundant check because `start_test` would make sure that `CHPL_HOME` is set.